staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -445,3 +448,33 @@ func validate(plugin *config.Preference) error {
 
 	return nil
 }
+
+// loadAliasesForHelpDisplay loads all aliases from the kuberc file and registers them
+// in the command hierarchy for help display. This is called when kubectl is run without arguments.
+func (p *Preferences) loadAliasesForHelpDisplay(rootCmd *cobra.Command, errOut io.Writer) {
+	kuberc, err := DefaultGetPreferences("", errOut)
+	if err != nil || kuberc == nil {
+		return
+	}
+
+	// Register aliases in the command
+	for _, alias := range kuberc.Aliases {
+		commands := strings.Fields(alias.Command)
+		existingCmd, _, err := rootCmd.Find(commands)
+		if err != nil {
+			continue
+		}
+
+		// Create a copy of the existing command with the alias name
+		aliasCmd := &cobra.Command{
+			Use:   alias.Name,
+			Short: fmt.Sprintf(i18n.T("Alias for: %s"), alias.Command),
```

Comment:
I don't think that this short form matches with the given output;

```
 
 Aliases from kuberc:
   kp            Alias for: kubectl get pods
   klogs         Alias for: kubectl logs
   kns           Alias for: kubectl get namespaces
   kdesc         Alias for: kubectl describe
```

from where `kubectl` is coming from?